sdk/monitor/Azure.Monitor.Query/tests/MonitorQueryTestEnvironment.cs

```diff
@@ -25,5 +26,36 @@ public class MonitorQueryTestEnvironment : TestEnvironment
         public string ConnectionString => GetRecordedOptionalVariable("CONNECTION_STRING");
         public string StorageAccountId => GetRecordedOptionalVariable("STORAGE_ID");
         public string StorageAccountConnectionString => GetRecordedOptionalVariable("STORAGE_CONNECTION_STRING");
+
+        public string GetAudience()
+        {
+            Uri authorityHost = new(AuthorityHostUrl);
+
+            if (authorityHost == AzureAuthorityHosts.AzurePublicCloud)
+            {
+                if (AzureAuthorityHosts.AzurePublicCloud.ToString() == LogsQueryAudience.AzurePublicCloud)
+                    return LogsQueryAudience.AzurePublicCloud.ToString();
+                if (AzureAuthorityHosts.AzurePublicCloud.ToString() == MetricsQueryAudience.AzurePublicCloud)
+                    return MetricsQueryAudience.AzurePublicCloud.ToString();
+            }
+
+            if (authorityHost == AzureAuthorityHosts.AzureChina)
```

Comment:
This should probably be an `else if`. No need to check for Azure China if Azure Public Cloud is being used.